util/uvmdvgen/sim_cfg.hjson.tpl

```diff
@@ -17,16 +17,20 @@
   // Testplan hjson file.
   testplan: "{proj_root}/hw/ip/${name}/data/${name}_testplan.hjson"
 
+% if has_ral:
   // RAL spec - used to generate the RAL model.
   ral_spec: "{proj_root}/hw/ip/${name}/data/${name}.hjson"
+% endif
 
   // Import additional common sim cfg files.
   // TODO: remove imported cfgs that do not apply.
 % if is_cip:
   import_cfgs: [// Project wide common sim cfg file
                 "{proj_root}/hw/dv/data/common_sim_cfg.hjson",
                 // Common CIP test lists
+% if has_ral:
                 "{proj_root}/hw/dv/data/tests/csr_tests.hjson",
+% endif
                 "{proj_root}/hw/dv/data/tests/mem_tests.hjson",
 % if has_interrupts:
                 "{proj_root}/hw/dv/data/tests/intr_test.hjson",
```

Comment:
intr test tests require ral model

Reply:
A similar question to before, does the ability to handle interrupts implicitly mean that the DUT has CSRs, since it seems that the DUT needs three interrupt specific CSRs to have interrupt capability?